source/opt/set_spec_constant_default_value_pass.h

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIBSPIRV_OPT_SET_SPEC_CONSTANT_DEFAULT_VALUE_PASS_H_
+#define LIBSPIRV_OPT_SET_SPEC_CONSTANT_DEFAULT_VALUE_PASS_H_
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include "module.h"
+#include "pass.h"
+
+namespace spvtools {
+namespace opt {
+
+// The optimization pass that sets the default values for the spec constants
+// that have SpecId decorations (i.e. those defined by
+// OpSpecConstant{|True|False} instructions).
+class SetSpecConstantDefaultValuePass : public Pass {
```

Comment:
Do we need to have the `Default` part in the name? It's just **fixing** the values of spec constants. Putting `Default` here let me feel like they are free to take some other value in some way.